### ISSUE TITLE
How to set custom header in SDWebImage 5.x

### ISSUE THREAD

**Issue by sumitpaytm** [SDWebImage/SDWebImage#2834]
In SdWebImage 4.x, I use below line to set header for image download
let manager = SDWebImageManager.shared.imageDownloader
manager?.setValue()

but this is not available in SdWebImage 5.x
How can I set it?

### New Issue Checklist

* [ ] I have read and understood the [CONTRIBUTING guide](https://github.com/rs/SDWebImage/blob/master/.github/CONTRIBUTING.md)
* [ ] I have read the [Documentation](http://cocoadocs.org/docsets/SDWebImage/)
* [ ] I have searched for a similar issue in the [project](https://github.com/rs/SDWebImage/issues) and found none

### Issue Info

 Info                    | Value                               |
-------------------------|-------------------------------------|
 Platform Name           | e.g. ios / macos / tvos / watchos
 Platform Version        | e.g. 12.0 / 10.14.0 / 12.0 / 5.0
 SDWebImage Version      | e.g. 5.0.0 / 4.4.0
 Integration Method      | e.g. carthage / cocoapods / manually
 Xcode Version           | e.g. Xcode 11 / Xcode 10
 Repro rate              | e.g. all the time (100%) / sometimes x% / only once
 Repro with our demo prj | e.g. does it happen with our demo project?
 Demo project link       | e.g. link to a demo project that highlights the issue

### Issue Description and Steps

Please fill in the detailed description of the issue (full output of any stack trace, compiler error, ...) and the steps to reproduce the issue.

**Comment by dreampiggy** [comment]
See the documentation and Wiki. Here is it:

[Use Request Modifier (5.0)](https://github.com/SDWebImage/SDWebImage/wiki/How-to-use#use-request-modifier-50)
